nwb_conversion_tools/datainterfaces/ecephys/ced/ceddatainterface.py

```diff
@@ -18,9 +26,12 @@ def get_source_schema(cls):
         source_schema["properties"]["file_path"].update(description="Path to CED data file.")
         return source_schema
 
-    def __init__(self, file_path: FilePathType):
-        super().__init__(file_path=file_path)
-
     @classmethod
     def get_all_channels_info(cls, file_path: FilePathType):
-        return cls.RX.get_all_channels_info(file_path)
+        """Retrieve and inspepct necessary channel information prior to initialization."""
```

Comment:
```suggestion
        """Retrieve and inspect necessary channel information prior to initialization."""
```

Reply:
I really need to find a spell check for linting in my IDE; every other type of format check has been supported so far, lol.